src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -56,6 +59,10 @@ interface Parameters extends FlowParameters {
         @Override
         public void execute(Parameters parameters) {
             Optional<Throwable> failure = parameters.getBuildResult().get().getFailure();
+            if (Paths.get("").toAbsolutePath().toString().toUpperCase().contains("ONEDRIVE")) {
```

Comment:
This is also not correct. There are gradle methods on Settings to get the location of the whole project. Those are what should be used.